content/en/docs/reference/glossary/probe.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Probe
+id: probe
+date: 2023-03-21
+full_link: /docs/concepts/workloads/pods/pod-lifecycle/#container-probes
+
+short_description: >
+  A diagnostic performed periodically by the kubelet on a container.
```

Comment:
I don't really like the use of word "diagnostic" as it often refers to troubleshooting rather than state or health. But cannot come up with short alternative for this. I'd be ok to keep it as is, but if anybody has an idea - please share

Reply:
```suggestion
  A check performed periodically by the kubelet on a container in a Pod.
```